test/kubemark/start-kubemark.sh

```diff
@@ -302,6 +302,10 @@ current-context: kubemark-context")
   metrics_mem_per_node=4
   metrics_mem=$((200 + ${metrics_mem_per_node}*${NUM_NODES:-10}))
   sed -i'' -e "s/{{METRICS_MEM}}/${metrics_mem}/g" "${RESOURCE_DIRECTORY}/addons/heapster.json"
+  metrics_cpu_per_node_numerator=${NUM_NODES:-10}
+  metrics_cpu_per_node_denumerator=2
```

Comment:
Ups, nothing like a copy paste name:) I'll fix it after it's merged to get it in quicker.